### DISCUSSION TITLE
Part 2 writeup questions

### DISCUSSION THREAD

**Discussion by shadowbourne** [wenceorg/comp4187#98] [Q&A] [answered]
In the events that one (myself and seemingly others by the looks of the discussion boards) cannot get part 2a to converge, What should we do about 2a questions, and all of 3b?  

As combined, this is worth 35% of the coursework, plus 15% for the 2a implementation which I assume we would receive a portion of the marks forward despite lack of full convergence.

Not being able to progress past 2a seems disproportionately detrimental/unfair to students, costing up to 50% of the marks for the CW, for what I assume is a minor error/lack of understanding in one part of the question.

Is there anything we can do to mediate this ? Like in a maths exam where you can still fully answer part b despite not having completed part a, as you are given a "correct starting point" to go off again. I imagine this is not possible in this course work, but is there anything else you could suggest? As currently obtaining high enough of a mark in the remainder of the course work to attain a pass overall is a worrying prospect for me.

Thankyou for your help.

**Comment by ovsen** [comment]
I agree with all points made above. Looking at previous years module grading, it seem quite scary that 17% of students failed the module.

I would like to add that this is a 10 credit module with 2 courseworks, the first one taking up 1 week to complete and the second has taken close to two weeks now. The amount of work seems very disproportionate.

**Reply by annereinarz** [reply]
We've run a very similar coursework last year and the reported time spent seemed exactly in line with what we expected. I'll get the numbers from everyone again this year and of course we'll adjust correspondingly moving forward.
I'll also definitely take into account that most seem to be struggling with part 2 when I mark. For most of the problems people are describing there is most likely an issue in only one component (e.g. boundary conditions, initial condition, time-step). In a case like that you'll receive marks for the components that do work.

**Comment by annereinarz** [comment, marked as answer]
For all of the coursework the write-up is to some extent marked separately from the implementation. So if you have an unexpected result but you document it correctly, e.g. plotting it and explaining the convergence rate and actually expected result, then you'll still get full marks for the write-up.
If you're stuck on 2a, at this point, I'd recommend moving on to part 3. The multigrid solver should give the same results as the explicit Euler, so as with the write-up, if its an error that stems from part 2a don't worry about it. I'll mark the multigrid separately from that error. If the problem isn't an unexpected convergence rate but a completely nonsensical result, implement the multigrid and test it with a different equation of your choice, I'd recommend just setting k=1 and adjusting the rhs.
